docker/files/freesurfer7.3.2-exclude.txt

```diff
@@ -355,13 +384,16 @@ freesurfer/bin/mri_fdr
 freesurfer/bin/mri_fieldsign
 freesurfer/bin/mri_fit_bias
 freesurfer/bin/mri_fslmat_to_lta
+freesurfer/bin/mri-func2sph
+freesurfer/bin/mri-funcvits
 freesurfer/bin/mri_fuse_intensity_images
 freesurfer/bin/mri_gca_ambiguous
 freesurfer/bin/mri_gcab_train
 freesurfer/bin/mri_gdfglm
 freesurfer/bin/mri_glmfit
 freesurfer/bin/mri_glmfit-sim
 freesurfer/bin/mri_gradient_info
+freesurfer/bin/mri_gradunwarp
```

Comment:
This could come in handy if we ever decide to incorporate https://github.com/nipreps/fmriprep/issues/1550

Reply:
Did you want to include it now, or is this just a note to self?

Reply:
Let's include it, I doubt it's big enough to matter

Reply:
In which case, since we'll be rebuilding nearly from scratch, I can bump down to v0.